src/main/kotlin/no/nav/familie/ef/sak/forvaltning/GjennoprettOppgavePåBehandlingTask.kt

```diff
@@ -0,0 +1,101 @@
+package no.nav.familie.ef.sak.forvaltning
+
+import no.nav.familie.ef.sak.behandling.BehandlingService
+import no.nav.familie.ef.sak.behandling.domain.Behandling
+import no.nav.familie.ef.sak.behandling.domain.BehandlingStatus
+import no.nav.familie.ef.sak.behandling.domain.BehandlingStatus.FATTER_VEDTAK
+import no.nav.familie.ef.sak.behandling.domain.BehandlingStatus.FERDIGSTILT
+import no.nav.familie.ef.sak.behandling.domain.BehandlingStatus.IVERKSETTER_VEDTAK
+import no.nav.familie.ef.sak.behandling.domain.BehandlingStatus.OPPRETTET
+import no.nav.familie.ef.sak.behandling.domain.BehandlingStatus.SATT_PÅ_VENT
+import no.nav.familie.ef.sak.behandling.domain.BehandlingStatus.UTREDES
+import no.nav.familie.ef.sak.infrastruktur.exception.feilHvis
+import no.nav.familie.ef.sak.oppgave.OppgaveRepository
+import no.nav.familie.ef.sak.oppgave.OppgaveService
+import no.nav.familie.ef.sak.oppgave.TilordnetRessursService
+import no.nav.familie.kontrakter.felles.oppgave.OppgavePrioritet
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import no.nav.familie.kontrakter.felles.oppgave.StatusEnum.FEILREGISTRERT
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+import java.util.Properties
+import java.util.UUID
+
+typealias EFOppgave = no.nav.familie.ef.sak.oppgave.Oppgave
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = GjennoprettOppgavePåBehandlingTask.TYPE,
+    maxAntallFeil = 1,
+    settTilManuellOppfølgning = true,
+    beskrivelse = "Finn og logg metadata for oppgave knyttet til behandling",
+)
+class GjennoprettOppgavePåBehandlingTask(
```

Comment:
Skulle vi hatt en test eller to her? Ikke opprett hvis feil status og opprett med verifisering av beskrivelse?

Reply:
Jepp - fikser!